monkey/monkey_island/cc/services/config.py

```diff
@@ -148,6 +163,18 @@ def update_config(config_json, should_encrypt):
         logger.info('monkey config was updated')
         return True
 
+    @staticmethod
+    def keep_PBA_files(config_json):
```

Comment:
rename this function to `set_config_pba_files` or something. Having a function with the current name and another function `remove_PBA_files` makes you think they do the opposite of one another, while they're not